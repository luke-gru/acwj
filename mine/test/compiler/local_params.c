void printsum(int a, int b) {
    int c;
    c = 10;
    printint(a);
    printint(b);
    printint(c);
    printint(a+b);
}

int main() {
    printsum(1000, 2000);
    return (0);
}
