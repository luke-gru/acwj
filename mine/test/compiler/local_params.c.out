1000
2000
10
3000
